README.md

```diff
@@ -511,9 +544,7 @@ h('div', [
 ]);
 ```
 
-## Helpers
-
-### SVG
+## SVG
```

Comment:
I'm not sure of the value or correctness of placing SVG and thunks under Helpers. So I removed Helpers. Is this cool?